reccmp/isledecomp/cvdump/types.py

```diff
@@ -201,7 +201,7 @@ class CvdumpTypesParser:
 
     # LF_CLASS/LF_STRUCTURE name and other info
     CLASS_NAME_RE = re.compile(
-        r"^\s+Size = (?P<size>\d+), class name = (?P<name>(?:[^,]|,\S)+)(?:, UDT\((?P<udt>0x\w+)\))?"
+        r"^\s+Size = (?P<size>\d+), class name = (?P<name>(?:[^,]|,\S)+)(?:, unique name = [^,]+)?(?:, UDT\((?P<udt>0x\w+)\))?"
```

Comment:
I reversed this in my microsoft-pdb fork:
https://github.com/madebr/microsoft-pdb/blob/1545c8af664f3a898356427eae86c610498d55e3/cvdump/type7.cpp#L672